README.md

```diff
@@ -3,7 +3,8 @@
 # Book store
```

Comment:
- [ ] Please kindly add steps to clone and run your project locally. This will help non technical users and some of them are potential recruiters.

Reply:
Added steps for cloning